org.kde.drydock/contents/ui/main.qml

```diff
@@ -127,6 +148,14 @@ Item {
                         running: true
                         visible: isBusy(model.Id)
                     }
+
+                    PlasmaComponents.Button{
+                        enabled: !(isBusy(model.Id) || (model.State == "exited"))
+                        iconSource: "bash-symbolic"
```

Comment:
I don't see the icon when running your changes locally, can I suggest we use a different one? Maybe `utilities-terminal` ?
```suggestion
                        iconSource: "utilities-terminal"
```